js/oyst/oneclick.js

```diff
@@ -11,10 +11,18 @@
 /**
  * OneClick Customs JS
  */
+"use strict";
 
 function showMessage(txt, type) {
-    var html = '<ul class="messages"><li class="' + type + '-msg"><ul><li>' + txt + '</li></ul></li></ul>';
+    var html = "<ul class=\"messages\"><li class=\"" + type + "\"-msg\"><ul><li>" + txt + "</li></ul></li></ul>";
```

Comment:
before the change, it looked clearer to me

Reply:
Yes, but with "use strict", I have to use " and not ' to surround code.

Reply:
And you can't use template strings I guess ? 
```
var html = `
  <ul class="messages">
    <li class="${type}-msg">
      <ul>
        <li>${txt}</li>
      </ul>
    </li>
   </ul>
`
```

Reply:
weirdo
http://ecma262-5.com/ELS5_HTML.htm#Section_14.1

```
A Use Strict Directive is an ExpressionStatement in a Directive Prologue whose StringLiteral is either the exact character sequences "use strict" or 'use strict'. A Use Strict Directive may not contain an EscapeSequence or LineContinuation.
```